_docs/introduction/quick_start.html

```diff
@@ -0,0 +1 @@
+<meta http-equiv="refresh" content="0; url=http://localhost:4000/docs/install/quickstart" />
```

Comment:
You probably want to redirect to http://docs.storageos.com/docs/install/quickstart